static/usr/lib/systemd/system/console-conf@.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Ubuntu Core Firstboot Configuration %I
+After=systemd-user-sessions.service plymouth-quit-wait.service
+After=rc-local.service
+# on core18/core20 the firstboot seeding happens via the core[18].start-snapd service,
+# make sure to start after so that the 'snap' command is available
+After=core18.start-snapd.service core.start-snapd.service
+# on core20 the user may invoke a recovery chooser, make sure the detection
+# service runs before
+After=snapd.recovery-chooser-trigger.service
+IgnoreOnIsolate=yes
+ConditionPathExists=/dev/tty0
+ConditionPathExists=|!/var/lib/console-conf/complete
+ConditionPathExists=|/run/snapd-recovery-chooser-triggered
+StartLimitInterval=0
+Conflicts=getty@%i.service
+
+[Service]
+Environment=PYTHONPATH=/usr/share/subiquity
+ExecStartPre=/bin/systemctl stop getty@%i
```

Comment:
This is not right. We have to use conflicts. I had opened https://github.com/snapcore/core-base/pull/84/files, but then decided to push it upstream. Which was ignored https://github.com/canonical/subiquity/pull/1478.

If we have to have the static files here. We should remove this.

Reply:
Let me double check that, with conflicts=getty@..I was not able to successfully transition from console-conf to getty on the first run of a UC console-conf-less image which ended up being console-conf & getty restarting in a loop